src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5761,18 +5791,31 @@ static void addSlur(const Notation& notation, SlurStack& slurs, ChordRest* cr, c
             if (cr->isGrace()) {
                 newSlur->setAnchor(Spanner::Anchor::CHORD);
             }
-            if (lineType == "dotted") {
-                newSlur->setStyleType(SlurStyleType::Dotted);
-            } else if (lineType == "dashed") {
+            if (lineType == "dashed") {
                 newSlur->setStyleType(SlurStyleType::Dashed);
+            } else if (lineType == "dotted") {
+                newSlur->setStyleType(SlurStyleType::Dotted);
+            } else if (lineType == "solid") {
```

Comment:
here trest an empty string the same
```cpp
} else if (lineType="" || lineType == "solid") {
```
Or just rely on `lineType's initialisation further up